### ISSUE TITLE
What is localhost 8883?

### ISSUE THREAD

**Issue by marcelstoer** [thomasnordquist/MQTT-Explorer#397]
Connecting to m20.cloudmqtt.com:22703 over TLS reports a refused connection to _localhost 8883_? What is that? What is expected to run there?

![localhost_8883](https://user-images.githubusercontent.com/624195/86527031-264cd800-be9b-11ea-82cb-74c6c0bb8511.jpg)

**Comment by Hello-Hyman** [comment]
Encountered the same problem, check the domain name you entered, there should be a space before the letter 'm20.xxx.xx'.

**Comment by marcelstoer** [comment]
Indeed, thanks. If you know where to look in my screen shot it's very clear: top-center in the dialog there's the full URL, showing the extra space.

Still, very odd the client would try to connect to localhost 8883 in this case.
